feat(tagCount):  Added tag count component. (#54)
patternfly/react-component-groups: packages/module/patternfly-docs/content/extensions/component-groups/examples/TagCount/TagCount.md

---
# Sidenav top-level section
# should be the same for all markdown files
section: extensions
subsection: Component groups
# Sidenav secondary level section
# should be the same for all markdown files
id: Tag count
# Tab (react | react-demos | html | html-demos | design-guidelines | accessibility)
source: react
# If you use typescript, the name of the interface to display props for
# These are found through the sourceProps function provided in patternfly-docs.source.js
propComponents: ['TagCount']
---
import TagCount from '@patternfly/react-component-groups/dist/dynamic/TagCount';

The **tag count** component generates a tag icon that displays a number, which represents a count value. 

> display as disable with no value.

Reviewer: ```suggestion
```

---
# Sidenav top-level section
# should be the same for all markdown files
section: extensions
subsection: Component groups
# Sidenav secondary level section
# should be the same for all markdown files

> id: Tag Count

Reviewer: ```suggestion
id: Tag count
```

# Tab (react | react-demos | html | html-demos | design-guidelines | accessibility)
source: react
# If you use typescript, the name of the interface to display props for
# These are found through the sourceProps function provided in patternfly-docs.source.js
propComponents: ['TagCount']
---

import TagCount from '@patternfly/react-component-groups/dist/dynamic/TagCount';

> The **TagCount** component generates a tag with count value.  If no count value is provided then the tag count will

Reviewer: ```suggestion
The **tag count** component generates a tag icon that displays a number, which represents a count value. 
```

display as disable with no value.

## Examples

> ### Tag count disabled
> 
> This example shows a tag count with no value. When no value is provided the tag count will display as disabled.

Reviewer: ```suggestion
### Basic tag count
A basic tag count uses the `count` property to display the number of tagged items.

```

Reviewer: would swap the example order to lead with the basic/expected version

> ```js file="./TagCountDisabledExample.tsx"

Reviewer: ```suggestion
```js file="./TagCountExample.tsx"
```

```

> ### Tag count with a value
> 
> This example shows a tag count with a value.

Reviewer: ```suggestion
### Disabled tag count

When no value is provided to a tag count component, it will be styled as disabled.

```

> ```js file="./TagCountExample.tsx"

Reviewer: ```suggestion
```js file="./TagCountDisabledExample.tsx"
```
